docs/analysis/datadog.md

```diff
@@ -41,6 +41,37 @@ stringData:
 
 `apiVersion` here is different from the `apiVersion` from the Datadog configuration above.
 
+!!! important
+    ###### Namespaced secret
+
+    To use a secret within the Analysis template namespace, add a secretRef section. 
```

Comment:
We need to add some logic explaining how credentials are retrieved:
1. if configured, try the secret referenced in the object
2. if not found (or invalid?), fallback to the global secret
3. if not found (or invalid?), fallback to arago-rollouts environment variables

Reply:
Done, let me know what you think.